apps/admin/src/components/notice/NoticePost/NoticePost.tsx

```diff
@@ -0,0 +1,85 @@
+import { color, font } from '@maru/theme';
+import { Button, Column } from '@maru/ui';
+import { flex } from '@maru/utils';
+import { ChangeEventHandler, useRef, useState } from 'react';
+import styled from 'styled-components';
+import { useNoticePostAction } from './NoticePost.hooks';
+
+const NoticePost = () => {
+    const contentTextareaRef = useRef<HTMLTextAreaElement>(null);
+    const [noticeData, setNoticeData] = useState({
+        title: '',
+        content: '',
+    });
+
+    const { handlePostButtonClick } = useNoticePostAction(noticeData);
+
+    const handleNoticeDataChange: ChangeEventHandler<HTMLInputElement | HTMLTextAreaElement> = (
+        event,
```

Comment:
```suggestion
        e
```
모든 event들은 e로 쓰이고있습니당

Reply:
ㄷㄷ